### PR TITLE
Support computation of network pressures in networks with multiple roots

### PR DESCRIPTION
**Pull request by vkip**
Depends on https://github.com/OPM/opm-common/pull/3702

### CODE REVIEW

**Comment by vkip** [comment]
jenkins build this opm-common=3702 please

**Comment by vkip** [comment]
https://github.com/OPM/opm-tests/pull/1082

File: opm/simulators/wells/WellGroupHelpers.cpp [hunk 1]
```diff
@@ -788,120 +788,124 @@ namespace WellGroupHelpers
...
-            root_to_child_nodes.push_back(node);
-            auto branches = network.downtree_branches(node);
-            if (branches.empty()) {
-                leaf_nodes.insert(node);
-            }
-            for (const auto& branch : branches) {
-                children.push(branch.downtree_node());
+        std::map<std::string, double> node_pressures;
+        auto roots = network.roots();
+        for (const auto& root : roots) {
```
Comment (GitPaean) [inline opm/simulators/wells/WellGroupHelpers.cpp:793]: If I did not see the code wrong, the following code is the only change, right?
```c++
        auto roots = network.roots();
        for (const auto& root : roots) {
```

Comment (vkip) [reply opm/simulators/wells/WellGroupHelpers.cpp:793]: Hi, yes, that is correct. Sorry, should have commented on that, didn't realize the diff was so messy.

**Comment by GitPaean** [comment]
The upstream PR is merged. I am getting this PR in now.

**Comment by GitPaean** [comment]
The regression tests should be added in a follow-up PR.

**Comment by vkip** [comment]
> The regression tests should be added in a follow-up PR.

Thanks, will do.

**Pull request merged**